Corona-Warn-App/src/main/assets/privacy_en.html

```diff
@@ -1256,7 +1256,7 @@ <h2>12. How can you withdraw your consent?
 
 </p>
 <h3>
-    a. Consent to "exposure logging"
+    a. Consent to 'exposure logging'
```

Comment:
Why did you change this from `""` to `''`? For the other points, it is `“”`. See https://github.com/corona-warn-app/cwa-documentation/issues/610.